### ISSUE TITLE
turn off note amount adjustment

### ISSUE THREAD

**Issue by RustoMCSpit** [thetwom/Tuner#47]
In the bottom window, the amount of visible notes can jump drastically. There should be an option to set how many notes are visible / an octave mode that after hearing the first new note it will show 6 notes above it and 6 below. This is a general idea, tweak at will

**Comment by thetwom** [comment]
Thanks for the feedback. At the moment, the strategy when finding a new target note is to first extend the visible domain and then zoom in to the target note. Is this what you mean? So you would prefer to not extend the visible range too much?

**Comment by RustoMCSpit** [comment]
I would prefer being able to lock how much the range shows and then lock it, I find the zooming in and out absolutely jarring. I would like if I could set it to say 'shows 12 notes' and then when it was near the note I want I press the tuner screen and it locks to that position.

**Comment by thetwom** [comment]
I guess, it is difficult to show a screen recording of some typical way of how you are using the app? It strongly depends on the use case how well the app works, so it would help me to understand better, why the current behaviour is annoying for you.

Regarding the request to lock to a position, this is already possible. Just press on the screen. You can also manually zoom and drag, but this does not solve your original issue.

**Comment by RustoMCSpit** [comment]
Yeah I know locking the screen is possible. I want the tuner to stop zooming in and out and I want to set how many notes (the zoom level) it shows at a time

**Comment by thetwom** [comment]
Thanks for your answer. For bringing changes to main version, it is crucial to me to understand why the current behaviour is non-ideal. Otherwise I will just follow the preferences of users without having sustainable improvements. So please help me to understand what leads to these drastic jumps and why they are annoying (if the target note switches drastically, we must of course somehow handle this).

**Comment by RustoMCSpit** [comment]
> Thanks for your answer. For bringing changes to main version, it is crucial to me to understand why the current behaviour is non-ideal. Otherwise I will just follow the preferences of users without having sustainable improvements. So please help me to understand what leads to these drastic jumps and why they are annoying (if the target note switches drastically, we must of course somehow handle this).

it isnt 'non-ideal', it's a preference. i find the zooming in and out to be nauseating and hard on the eyes especially when the tuner is going on a frenzy. id rather just have it jammed to 12 note octaves, (or any other note amount) not zooming in or out. it also lets me see more easily my melodies as i sing for long periods of time

**Comment by RustoMCSpit** [comment]
any update on this?

**Comment by thetwom** [comment]
No update here. I am still trying to understand the use-case behind this request, if it fits to a tuner and if it doesn't whether it is worth to implement such an option. 

To me it sounds, that you would mainly like the tuner to stop zooming in and out, but rather see melodies which are played in the timeline. While this contradicts a bit with a tuner, I understand that following melodies is a valid use case. But isn't the manual zooming option, what you need. If we restrict the automatic zooming to larger note ranges, it will still "focus" to the current target note somehow. 

I understand, that it is a matter of preference, how the tuner behaves. The problem is that I cannot start following individual preferences since they might contradict each other or it would lead to more and more preferences which at some time would make it completely overloaded. So the question is, if your feature request can improve the app as a whole, where many people benefit from.

**Comment by RustoMCSpit** [comment]
> To me it sounds, that you would mainly like the tuner to stop zooming in and out, but rather see melodies which are played in the timeline

i want to be able to lock the tuners bottom screen to octaves only, notes do not zoom in or out and if the tuner goes to a new octave the letters will indicate it. so it always starts on a and ends on the a in the next octave

**Comment by thetwom** [comment]
What I still don't understand, why manual dragging and zooming in the bottom window isn't doing the job. When you have manually adjusted your view it won't focus or move but stick to your defined range. This feature is exactly meant for such use cases.

**Comment by RustoMCSpit** [comment]
> What I still don't understand, why manual dragging and zooming in the bottom window isn't doing the job. When you have manually adjusted your view it won't focus or move but stick to your defined range. This feature is exactly meant for such use cases.

defined range changes octaves, i want it to move with the a note at the botton and top across octaves automatically. the zooming in and out makes me sick

**Comment by thetwom** [comment]
So in the end the request is to show not two one note below and above the target, but have it flexible? And if making the range large enough, it would not zoom anymore, but still center automatically around the target note?

**Comment by RustoMCSpit** [comment]
> So in the end the request is to note show one note below and one above the target, but have it flexible? And if making the range large enough, it would not zoom any more, but still centre automatically around the target note?

show a whole octave always, always start on a at the bottom and a at the top, no zooming in or out
